force-app/main/default/classes/RD2_RecurringDonation.cls

```diff
@@ -159,6 +159,15 @@ public inherited sharing class RD2_RecurringDonation {
         return this;
     }
 
+    /**
+    * @description Returns True if the Next Payment Date has changed on an Elevate connected RD
+    * @return Boolean True if the Next Payment Date has changed on an Elevate connected RD
+    */
+    public Boolean needsNextPaymentScheduleCheck() {
+        return changedFields.contains(npe03__Recurring_Donation__c.npe03__Next_Payment_Date__c)
```

Comment:
Since RD2 disables the CRLP for this field, I think this is a safe way to "know" that a Payment has been Paid?